### PR TITLE
Fix inline_svg helper method for middleman

### PR DESCRIPTION
**Pull request by vitaliiprodan**
Why:

* To allow deploy middleman projects on heroku
* Old code threw error on Heroku

How:

* Updated `file_path` variable

### CODE REVIEW

File: app/helpers/front_man/images_helper.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@ def inline_svg(filename)
       when :rails
         file_path = "app/assets/images/#{filename}"
       when :middleman
-        file_path = sitemap.find_resource_by_path(image_path('icons.svg'))
-          .source_file
+        file_path = "#{Middleman::Application.root}/source/images/#{filename}"
+        File.read(file_path)
```
Comment (vfonic) [inline app/helpers/front_man/images_helper.rb]: This line is unnecessary.

Comment (vitaliiprodan) [inline app/helpers/front_man/images_helper.rb]: Right. Also I was wrong with `Middleman::Application.root`. It throws an error in Rails aplications

Comment (vitaliiprodan) [inline app/helpers/front_man/images_helper.rb]: In general this PR is unnecessary. Sorry :)

Comment (vfonic) [inline app/helpers/front_man/images_helper.rb]: Np!
I expected `FrontMan.framework` together with `when :middleman` would prevent Rails from ever executing those lines of code...hmmm

**Pull request closed without merging**